docs/wallets/helium-wallet-app.mdx

```diff
@@ -180,9 +187,8 @@ are requesting or add a note.
 
 ## Helium Identity
 
-Use the Helium Wallet app to connect to dApps and authenticate your identity on the Solana
-blockchain. Using the in-app browser, access dApps like Jupiter Aggregator or
-[stake and vote your tokens using Realms](/governance/realms).
+Use the Helium Wallet App to connect to dApps and authenticate your identity on the Solana
+blockchain. Using the in-app browser, access dApps such as Jupiter Aggregator or Orca.
```

Comment:
We should call out modular governance as the update to Realms instead of replacing with Orca.

Reply:
I just added the modular governance link, since there is no page about Modular Goverance on docs yet.